docs/docs/deployment/post_deployment/egress_store_approval_process.md

```diff
@@ -0,0 +1,86 @@
+---
+id: egress_store_approval_process
+title: Exporting data from AppStream-enabled workspaces
+sidebar_label: Exporting data from AppStream-enabled workspaces
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+As a researcher, if you have AppStream and Egress store enabled, AppStream provides limited access to the workspace that is being used. You can view and use data within that workspace. When you work on this data, it can be saved in the Egress folder, and then a request can be made to export that data. But if you want to export data from that workspace, it requires approval. 
+
+In the following example, you don’t currently have access to the `hello.txt` file out of your workspace. Hence, you need permission to export this file. The **Submit Egress Request** button shown in the figure below is used to export data from your workspace. 
+
+ <img src={useBaseUrl('img/deployment/post_deployment/e1.png')} />
```

Comment:
Let's move the picture files to `/reference/egress1.png`. Same with the other 2 screenshots.

Reply:
Fixed all the three images.